app/site/_includes/project-card.liquid

```diff
@@ -0,0 +1,49 @@
+<li class="usa-card project-card tablet:grid-col-12" org-name="{{ repo.owner }}">
+    <div class="usa-card__container">
+      <div class="usa-card__header">
+        <h2 class="usa-card__heading">
+          <a href={{ repo.link | url }} class="text-no-underline">{{ repo.name }}</a>
```

Comment:
@IsaacMilarky Could you add this field to all repo data json files:
link: "/{repo_owner}/{repo_name}/"

Reply:
I think it would be better to parse that string through the liquid template since repo.owner and repo.name are existing variables already. I pushed a commit for this change for you to review.

Reply:
Thanks for flagging. I was able to make it work on the frontend! Adding this field to all repo data json files is no longer needed